datajoint/erd.py

```diff
@@ -121,6 +121,18 @@ def __init__(self, source, context=None):
                     if node.startswith('`%s`' % database):
                         self.nodes_to_show.add(node)
 
+            for full_table_name in self.nodes._nodes.keys():
+                #one entry per parent
+                parents = connection.dependencies.parents(full_table_name,True)
+                
+                #total count of all the foreign primary keys
+                foreign_primary_keys = sum(len(parent['attr_map']) for parent in parents.values())
```

Comment:
foreign keys may have overlapping sets of attributes, so it's not enough to count them.  Use a set to enumerate all fo them.

Reply:
https://github.com/datajoint/datajoint-python/blob/d4254e78f306bbaad865adc0969e09ee9674c788/datajoint/erd.py#L137